functions-python/helpers/database.py

```diff
@@ -18,15 +18,69 @@
 import threading
 from typing import Final
 
-from sqlalchemy import create_engine, text
-from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine, text, event
+from sqlalchemy.orm import sessionmaker, mapper, class_mapper
 import logging
 
+from database_gen.sqlacodegen_models import Feed, Gtfsfeed, Gtfsrealtimefeed, Gbfsfeed
+
 DB_REUSE_SESSION: Final[str] = "DB_REUSE_SESSION"
 lock = threading.Lock()
 global_session = None
 
 
+def configure_polymorphic_mappers():
```

Comment:
These functions should be moved to a shared folder once we have the ability to share code between API and Python functions. Pending completion of #779 and #826